OrbitLinuxTracing/TracerThread.cpp

```diff
@@ -518,8 +541,8 @@ void TracerThread::ProcessSampleEvent(const perf_event_header& header,
     ++stats_.uprobes_count;
 
   } else if (is_gpu_event) {
-    int32_t common_type = ReadTracepointCommonType(ring_buffer);
-    LOG("Received GPU event with common type: %d", common_type);
+    auto event = ConsumeSampleRaw(ring_buffer, header);
```

Comment:
Actually, one more thing, I think we should defer the processing of the gpu event if possible, otherwise processing might cause other buffers to overflow.  If there aren't that many gpu events, it shouldn't be too bad, but ideally we'd follow the same pattern as for uprobe events (using DeferEvent(...)).

Reply:
If there are not too many events generated per frame, let's add a "TODO".  Also I think there are currently some logs being printed out that might slow down the processing.

Reply:
Added a TODO, as I want to have a working version end-to-end asap. I'll look this once we have this feature working.

Reply:
Btw, there are ~20 events per frame typically.

Reply:
Ah thanks, that's fine for now then.